lib/index-basic.js

```diff
@@ -10,9 +10,6 @@
 
 var Plotly = require('./core');
 
-Plotly.register([
-    require('./bar'),
-    require('./pie')
-]);
+Plotly.register([require('./bar'), require('./pie')]);
```

Comment:
Is is possible to configure `prettier` to respect vertical formatting?

I see the need to split long lines, but joining short one into a single line doesn't help here.